webpack/assets/javascripts/react_app/components/HostDetails/Tabs/index.js

```diff
@@ -1,13 +1,15 @@
 import React from 'react';
+import { translate as __ } from '../../../common/I18n';
 import { addGlobalFill } from '../../common/Fill/GlobalFill';
-import { DEFAULT_TAB } from '../consts';
+import { DEFAULT_TAB, TABS_SLOT_ID } from '../consts';
 import OverviewTab from './Overview';
 
 export const registerCoreTabs = () => {
   addGlobalFill(
-    'host-details-page-tabs',
+    TABS_SLOT_ID,
     DEFAULT_TAB,
     <OverviewTab key="host-details-overview-tab" />,
-    1000
+    1000,
+    { translations: { overview: __('Overview') } }
```

Comment:
I might be missing something, why not use?
```suggestion
    { title: __('Overview') }
```

Reply:
a slot mostly has multiple fills,  while the [selector](https://github.com/theforeman/foreman/pull/8695/files#diff-1d486cf5c655a2d35d078eb58bc7348ccc80d2fc262e6849f69914f22a9c7bfaR43) returns the entire slot metadata. using the same key ('title') across fills will override its value.

Reply:
When would a fill need to modify the slot? Shouldn't the metadata be specific for the fill (like the weight is)? If I look at the snapshots, looks like the metadata is stored on each fill separately, so we could pull it out separately as well instead of merging it into one object on the slot.

Reply:
When a slot renders, it selects all of its fills sorted by weight via `selectComponentByWeight` selector.
The same as with the metadata, each fill has metadata, but a slot aggregates its fills metadata via `selectSlotMetadata`.
 I'm not sure why we would pull the metadata out separately per fill?

Reply:
Because the metadata is specific to the fill, so why is it aggregated by the slot?

Reply:
a specific fill cannot be rendered by itself, it needs its slot,  slots render their fill list. That's why the metadata is aggregated when a slot renders, we can store the metadata in the slot instead:
`Slot A -> {[ fill "1", fill "2" ], metadata}`

Reply:
@tbrisker, do you have any concerns?